management_api_app/tests/test_api/conftest.py

```diff
@@ -16,10 +16,21 @@ def no_database():
                     yield
 
 
+def override_get_user():
```

Comment:
are you then running all tests with admin role? 

We should specifically write atleast one test for each route that privileged route should not execute using non privileged user.

Reply:
Created #340 for this purpose